app/views/account/utility_providers/new.html.slim

```diff
@@ -0,0 +1,29 @@
+.container.pt-3
+  .jumbotron
+    .col-8
+      h1.text-left
+        = t('.add-utilities-provider')
+      h2.text-left
+        = t('.about-company')
+      .row
+        .col-md-9.pl-0
+          p
+            = t('.name-of-company')
```

Comment:
it look weird on UI i think just Company name in bold should be enough
Name of company Friesen Inc -> **Friesen Inc**